static/show-me/browserwindow/main.js

```diff
@@ -23,15 +23,27 @@ app.on('ready', () => {
       y: 200
     })
   )
-
-  // A transparent window
-  windows.push(
-    new BrowserWindow({
-      transparent: true,
-      x: 300,
-      y: 300
-    })
-  )
+  
+  // A transparent window. On Windows OS, a transparent window must be frameless
+  if (process.platform != 'win32') {
+    windows.push(
+      new BrowserWindow({
+        transparent: true,
+        x: 300,
+        y: 300
+      })
+    )  
+  }
+  else {
+    windows.push(
+      new BrowserWindow({
+        transparent: true,
+        frame: false,
+        x: 300,
+        y: 300
+      })
+    )
+  }
```

Comment:
We want to keep it as simple as possible to maintain, so it would be best to minimize the amount of duplicate code. Additionally, for future reference, you generally want to start your if-else conditionals with the positive check.

```suggestion

  // A transparent window.
  const windowOptions ={
    transparent: true,
    x: 300,
    y: 300
  }
  // On Windows platforms, a transparent window must be frameless
  if (process.platform === 'win32') {
    windowOptions.frame = false
  }
  windows.push(new BrowserWindow(windowOptions))
```

Reply:
Thanks, this is much cleaner! Pushed new changes that include this.

Reply:
@malept I resolved this comment thread since I pushed the update, but that doesn't make the "Requested changes must be addressed" blocker go away. I reset your review status, hopefully if you approve the review that will clear out the "change requested" blocker?

Apologies for the confusion, I'm a little new to the GitHub PR workflow. In Azure DevOps PRs, marking a comment thread as "resolved" clears that request throughout the PR, so that's what I expected would happen here too. But evidently GitHub works a little differently.